truss/tests/test_build.py

```diff
@@ -1,67 +1,18 @@
-from pathlib import Path
-
 from truss.base.truss_spec import TrussSpec
-from truss.truss_handle.build import init, init_directory, load
+from truss.truss_handle.build import init_directory, load
 from truss_chains.deployment import code_gen
 
 
 def test_truss_init(tmp_path):
-    dir_name = str(tmp_path)
-    init(dir_name)
-    spec = TrussSpec(Path(dir_name))
+    spec = TrussSpec(init_directory(tmp_path))
     assert spec.model_module_dir.exists()
     assert spec.data_dir.exists()
     assert spec.truss_dir == tmp_path
     assert spec.config_path.exists()
 
 
-def test_truss_init_with_data_file_and_requirements_file_and_bundled_packages(tmp_path):
```

Comment:
Mentioned in this [PR](https://github.com/basetenlabs/truss/pull/1367), but this was the only use case of passing data_files / requirements_file / bundled_packages into `init`. Since no real code uses it, it's safe to delete the test while we delete `init` in favor of `init_directory`